src/utils/ast.ts

```diff
@@ -116,10 +116,17 @@ export function dig(node: ESTree.Node, identifier: string | RegExp): boolean {
     : false
 }
 
+export function isPage(node: ESTree.CallExpression) {
```

Comment:
I'm kind of dubious on whether this function is a good idea or not, since you have to check if `node.callee` is a `MemberExpression` anyway to get the type checking to work properly. Open to thoughts.

Reply:
How about instead accepting a `MemberExpression` and then doing `isPage(node.callee)`.